src/axom/core/tests/core_openmp_main.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "gtest/gtest.h"
+
+#include "axom/config.hpp"  // for compile-time definitions
+
+#include "core_openmp_map.hpp"
+
+int main(int argc, char** argv)
+{
+  ::testing::InitGoogleTest(&argc, argv);
+
+  // add this line to avoid a warning in the output about thread safety
+  ::testing::FLAGS_gtest_death_test_style = "threadsafe";
+
+#ifdef EIGEN_SOLVE_TESTER_SHOULD_SEED
```

Comment:
This seems like a copy/paste artifact.
``EIGEN_SOLVE_TESTER_SHOULD_SEED`` will never be true in this file.

If we should test this using seeds, we need a `#define` in this file that is set at the top of this file
e.g. `OMP_MAP_TESTER_SHOULD_SEED` ?

Reply:
+1.  Thanks; I missed this.